docs/xls-38d-cross-chain-bridge/ledger-object-types/xchainownedclaimid.md

```diff
@@ -61,4 +98,17 @@ An `XChainCreateClaimID` transaction is used to create a new `XChainOwnedClaimID
 | `XChainClaimID`           | `string`          | `UINT64`          | Yes       | The unique sequence number for a cross-chain transfer. |
 
 
+### XChainClaimAttestations
+
+| Field                         | JSON Type         | Internal Type | Required | Description |
+|-------------------------------|-------------------|---------------|----------|-------------|
+| `XChainClaimProofSig`         | `array`           | `OBJECT`      | Yes      | An attestation from one witness server. |
+| `Amount`                      | `Currency Amount` | `AMOUNT`      | Yes      | The amount to claim on the destination chain. |
```

Comment:
I like the version in the other file better, where this refers to the amount in the `XChainCommit` transaction. That's true of the `Destination` field as well.